openedx/core/djangoapps/discussions/models.py

```diff
@@ -0,0 +1,61 @@
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from jsonfield import JSONField
+from model_utils.models import TimeStampedModel
+from opaque_keys.edx.django.models import LearningContextKeyField
+from simple_history.models import HistoricalRecords
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+class LearningContextDiscussionConfig(TimeStampedModel):
+    """
+    Associates a learning context with discussion provider and configuration
+    """
+
+    context_key = LearningContextKeyField(
+        primary_key=True,
+        db_index=True,
+        unique=True,
+        max_length=255,
+        verbose_name=_("Learning Context"),
+    )
+    enabled = models.BooleanField(
+        default=True,
+        help_text=_("If disabled, the discussions in the associated learning context/course will be disabled.")
+    )
+    lti_configuration = JSONField(
+        blank=True,
+        default={},
+        help_text=_("The LTI configuration data for this context/provider."),
+        # - launch_url
+        # - api_key
+        # - api_secret
+    )
```

Comment:
That's the plan.  I think @stvstnfrd is still in process on the code, and the LtiConfiguration isn't ready to be used yet anyway.

Reply:
Yeah, I initially wired it up like that because I wasn't clear on what needed done to use the `LtiConfiguration` yet.

I have made this change now, though I'm not 100% clear on feature disparity...